spk/sonarr/Makefile

```diff
@@ -1,22 +1,19 @@
-SPK_NAME = nzbdrone
+SPK_NAME = sonarr
 SPK_VERS = $(shell date +%Y%m%d)
-SPK_REV = 20
+SPK_REV = 22
 SPK_ICON = src/sonarr.png
 
-# Mono not supported on ppc platforms. C.f. cross/mono/Makefile and references therein for details.
-UNSUPPORTED_ARCHS = $(PPC_ARCHS)
+DEPENDS = cross/sonarr cross/libstdc++ cross/sqlite
 
-DEPENDS = cross/curl cross/mediainfo cross/sqlite cross/sonarr
-
-SPK_DEPENDS = "mono>3.6"
+UNSUPPORTED_ARCHS = $(DOTNET_UNSUPPORTED_ARCHS)
 
 MAINTAINER = SynoCommunity
 DESCRIPTION = Sonarr is a PVR for newsgroup and torrent users. It can monitor multiple RSS feeds for new episodes of your favourite shows and will grab, sorts and rename them. It can also be configured to automatically upgrade the quality of files already downloaded if a better quality format becomes available.
 DESCRIPTION_FRE = Sonarr est un PVR pour les utilisateurs de groupes de discussion et torrents. Il peut surveiller plusieurs flux RSS pour les nouveaux épisodes de vos séries préférées et saisira, sortes et les renomme. Il peut également être configuré pour mettre à jour automatiquement la qualité des fichiers déjà téléchargés si un meilleur format de qualité devient disponible.
 DESCRIPTION_SPN = Sonarr es un PVR para los usuarios de grupos de noticias y torrents. Se puede controlar múltiples canales RSS para nuevos episodios de sus programas favoritos y se agarra, tipo y les cambia el nombre. También puede ser configurado para actualizar automáticamente la calidad de los archivos ya descargados si un formato de mejor calidad disponible.
 DISPLAY_NAME = Sonarr
```

Comment:
We must use different Display Names for nzbdrone and sonarr (internal package names). Otherwise there will be two different packages in the repository (and in DSM Package Manager) with the same name.